angular/src/app/index/index.component.html

```diff
@@ -0,0 +1,39 @@
+<header>
+  <img src="./assets/images/header.png" alt="Oupsie, this doesn't work" width="100%" height="100%">
```

Comment:
imo this header is way too tall, considering it doesn't even fit on my screen
it would be cool if you could ask emi to make something shorter that's more suitable as a header

Reply:
Weird, it fits perfectly on my screen, 1920x1080. But yes I could ask. Problem is that he / she (idk ?) already work a lot xD